test/matrix.cc

```diff
@@ -233,7 +233,7 @@ TEST(Matrix, test_matrix) {
 
   CostMatrix cost_matrix_abort_source;
   cost_matrix_abort_source.SourceToTarget(request, reader, mode_costing, sif::TravelMode::kDrive,
-                                          100000.0);
+                                          90000.0);
```

Comment:
this test is testing that we abort the search for anything over 100 km and tests that 1 pair wasn't connected before expanding that far. so it takes less cycles to find the pair that the test tries to miss here, that's why the test failed. it's good that it's even showing in the test.

disclaimer: I tried to reproduce this test on master but couldn't for some reason. I changed the config with the 100k m max_matrix_distance, it still found all 16 pairs and there was no change to this PR. I have 0 idea why..

Reply:
fwiw, the time was identical with `route` for that pair